bridge_common.spec

```diff
@@ -0,0 +1,67 @@
+%define pkg_name tendrl-bridge-common
+%define pkg_version 0.0.1
+%define pkg_release 1
+
+Name: %{pkg_name}
+Version: %{pkg_version}
+Release: %{pkg_release}%{?dist}
```

Comment:
Please drop the `%{pkg_*}` boilerplate and just use

```
Name: tendrl-bridge-common
Version: 0.0.1
Release: 1
```

So that this .spec file will work with other RPM ecosystem tools like `rpmdev-bumpspec`